src/snippets/plotsvy.html

```diff
@@ -1 +1 @@
-<div id='plotsvy-div'></div>
\ No newline at end of file
+<div id='plotsvy-div' style="height: 100%"></div>
```

Comment:
This id should really be a class type attribute not an id type attribute.  Then you could add the style changes to css instead of directly here.